packages/react-components/src/theme-provider/src/ThemeProvider.css

```diff
@@ -1,6 +1,191 @@
+    --foreground-color-3: var(--cloud-900);
+    --foreground-color-4: var(--neutral-dark-800);
+    --foreground-color-4-hover: var(--neutral-dark-300);
+    --foreground-color-4-active: var(--neutral-dark-100);
+    --foreground-color-5: var(--marine-600);
+    --foreground-color-5-hover: var(--marine-500);
+    --foreground-color-5-active: var(--marine-400);
+    --foreground-color-brand: var(--primary-500);
+    --foreground-color-brand-hover: var(--primary-400);
+    --foreground-color-brand-active: var(--primary-300);
+    --foreground-color-brand-light: var(--neutral-dark-800);
+    --foreground-color-critical: var(--alert-700);
+    --foreground-color-critical-hover: var(--alert-500);
+    --foreground-color-critical-active: var(--alert-400);
+    /* TEXT */
+    --primary-text: var(--static-white);
+    --primary-text-hover: var(--static-white);
+    --primary-text-active: var(--static-white);
+    --secondary-text: var(--cloud-300);
+    --secondary-text-hover: var(--cloud-100);
+    --secondary-text-active: var(--cloud-50);
+    --tertiary-text: var(--cloud-700);
+    --tertiary-text-hover: var(--cloud-600);
+    --tertiary-text-hover-active: var(--cloud-500);
+    --brand-text: var(--primary-500);
+    --brand-text-hover: var(--primary-400);
+    --brand-text-active: var(--primary-400);
+    /* ICON */
+    --primary-icon: var(--static-white);
+    --secondary-icon: var(--cloud-500);
+    /* BORDER */
+    --primary-border: var(--neutral-dark-600);
+    --primary-border-hover: var(--neutral-dark-400);
+    --primary-border-active: var(--neutral-dark-300);
+    --secondary-border: var(--neutral-dark-800);
+    --tertiary-border: var(--neutral-dark-400);
+    --quaternary-border: var(--marine-400);
+    --quaternary-border-hover: var(--marine-600);
+    --brand-border: var(--primary-700);
+    --critical-border: var(--alert-700);
+    --critical-border-hover: var(--alert-600);
+    --critical-border-active: var(--alert-500);
+    /* INFO */
+    --info-50: var(--marine-900);
+    --info-500: var(--marine-600);
+    --info-600: var(--marine-700);
+    --info-900: var(--marine-50);
+    /* SUCCESS */
+    --success-50: var(--green-900);
+    --success-500: var(--green-100);
+    --success-900: var(--green-50);
+    /* WARNING */
+    --warning-50: var(--sunray-900);
+    --warning-500: var(--sunray-600);
+    --warning-900: var(--sunray-50);
+    /* CRITICAL */
+    --critical-50: var(--alert-900);
+    --critical-500: var(--alert-600);
+    --critical-900: var(--alert-50);
+    --critical-text: var(--alert-700);
+    --critical-text-hover: var(--alert-600);
+    --critical-text-active: var(--alert-500);
+    /* STATIC */
+    --static-white: #FFFFFF;
+    /* SHADOWS */
+    --shadow-lightness: 3%;
+    --shadow-opacity: 1;
+    /* FOCUS */
+    --o-ui-focus-ring-color: var(--primary-800);
+}
+
 /* APRICOT */
-/* APRICOT | LIGHT */
-.o-ui-apricot-theme.o-ui-light-color-scheme {
+.o-ui-apricot-theme {
```

Comment:
you shouldn't remove `.o-ui-light-color-scheme`. Without a color scheme it shouldn't work.